data/channels.csv

```diff
@@ -13197,7 +13197,7 @@ KTVKDT1.us,KTVK-DT1,,,,US,US-AZ,Phoenix,s/US-AZ,eng,,FALSE,,,,http://www.azfamil
 KTVKDT2.us,KTVK-DT2,,Comet TV,,US,US-AZ,Phoenix,s/US-AZ,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/9/97/Comet_logo.svg/512px-Comet_logo.svg.png
 KTVKDT3.us,KTVK-DT3,,Circle,,US,US-AZ,Phoenix,s/US-AZ,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/e/e8/Circle_Network_logo.svg/512px-Circle_Network_logo.svg.png
 KTVKDT4.us,KTVK-DT4,,THIS,,US,US-AZ,Phoenix,s/US-AZ,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/1/12/This_TV_logo.svg/512px-This_TV_logo.svg.png
-KTVKhallikBilbait.kw,KTV Khallik Bilbait,,,,KW,,,c/KW,ara,,FALSE,,,,,https://i.imgur.com/nykv5J5.png
+KTVKhallikBilbait.kw,KTV Khallik Bilbait,,KTV Drama,,KW,,,c/KW,ara,,FALSE,,,,,https://i.imgur.com/krojJJ7.png

```

Comment:
Alternative channel names must be specified in the third column (https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#channels)

```suggestion
KTVKhallikBilbait.kw,KTV Khallik Bilbait,KTV Drama,,,KW,,,c/KW,ara,,FALSE,,,,,https://i.imgur.com/krojJJ7.png
```

Reply:
made a new commit with all the changes requested